src/gridstack.js

```diff
@@ -1296,7 +1296,7 @@
       // TODO: cellHeight = cellHeight() causes issue (i.e. remove strictCellHeight above) otherwise
       // when sizing up we jump almost right away to next size instead of half way there. Not sure
       // why as we don't use ceil() in many places but round() instead.
-      cellHeight = self.$el.height() / parseInt(self.$el.attr('data-gs-current-row'));
+      cellHeight = (self.$el.height() + grid.verticalMargin()) / parseInt(self.$el.attr('data-gs-current-row'));
```

Comment:
should be self.verticalMargin() but I can change it later.

Reply:
while this fixes the issue, one downside is that now you need to stretch WAY past 50% point to get next size up. So that needs to be looked at as well.